portfolio/src/main/webapp/index.html

```diff
@@ -4,15 +4,58 @@
     <meta charset="UTF-8">
     <title>My Portfolio</title>
     <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="bootstrap.min.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  
+  <!--Display Nav Bar at top of page.-->
+  <nav class="navbar navbar-expand-lg navbar-light bg-light">
+    <div class="collapse navbar-collapse" id="navbarColor03">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item active">
+          <a class="nav-link" href="#">HOME</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">ABOUT</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">PROJECTS</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+
+  <body onpageshow="slideShow()">
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Kira Pancha</h1>
+      <img src="/images/me.jpg" alt="Picture of Kira" width="150px">
+      <p>Hi there! My name is Kira, and I'm pursuing a B.S. in Computer Science at Georgia Tech.</p>
+
+      <!--Display slideshow.-->
+      <div id="image-slideshow-container">
+        <div class="mySlides">
+          <img src="/images/places-0.jpg" width="400px">
+        </div>
+        <div class="mySlides">
+          <img src="/images/places-1.jpg" width="400px">
+        </div>
+        <div class="mySlides">
+          <img src="/images/places-2.jpg" width="400px">
+        </div>
+        <div class="mySlides">
+          <img src="/images/places-3.jpg" width="400px">
+        </div>
+        <div class="mySlides">
+          <img src="/images/places-4.jpg" width="400px">
+        </div>
+        <div class="mySlides">
+          <img src="/images/places-5.jpg" width="400px">
+        </div>
+        <div class="mySlides">
+          <img src="/images/places-6.jpg" width="400px">
+        </div>
```

Comment:
If possible, consider adding `alt` text for these: https://google.github.io/styleguide/htmlcssguide.html#Multimedia_Fallback